docs/sql-programming-guide.md

```diff
@@ -184,20 +175,20 @@ showDF(df)
 </div>
 
 
-## DataFrame Operations
+## Untyped Dataset Operations
 
-DataFrames provide a domain-specific language for structured data manipulation in [Scala](api/scala/index.html#org.apache.spark.sql.DataFrame), [Java](api/java/index.html?org/apache/spark/sql/DataFrame.html), [Python](api/python/pyspark.sql.html#pyspark.sql.DataFrame) and [R](api/R/DataFrame.html).
+Datasets provide an untyped domain-specific language for structured data manipulation in [Scala](api/scala/index.html#org.apache.spark.sql.Dataset), [Java](api/java/index.html?org/apache/spark/sql/Dataset.html), [Python](api/python/pyspark.sql.html#pyspark.sql.DataFrame) and [R](api/R/DataFrame.html).
```

Comment:
this is pretty confusing because it makes it sound like in Python and R people should also program Datasets.